common/cgroup/cgroup.go

```diff
@@ -35,15 +35,25 @@ import (
 
 type addIsolatorFunc func(opts []*unit.UnitOption, limit *resource.Quantity) ([]*unit.UnitOption, error)
 
+const (
+	// The following const comes from
+	// #define CGROUP2_SUPER_MAGIC  0x63677270
+	// https://github.com/torvalds/linux/blob/v4.6/include/uapi/linux/magic.h#L58
+	CgroupfsMagicNumber = 0x63677270
```

Comment:
Could this be called cgroup2 instead of cgroup, since "cgroup" also exists and has a different value?

```
#define CGROUP_SUPER_MAGIC	0x27e0eb
#define CGROUP2_SUPER_MAGIC	0x63677270
```